templates/config/initializers/sentry.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
-if defined?(Raven)
-  Raven.configure do |config|
-    config.processors -= [Raven::Processor::PostData]
```

Comment:
This is no longer supported, nor needed: https://docs.sentry.io/platforms/ruby/migration/#removed-processors